go.mod

```diff
@@ -64,7 +64,7 @@ require (
 	github.com/k14s/imgpkg v0.17.0
 	github.com/k14s/kbld v0.31.0
 	github.com/k14s/semver/v4 v4.0.1-0.20210701191048-266d47ac6115
-	github.com/k14s/ytt v0.32.1-0.20210511155130-214258be2519
+	github.com/k14s/ytt v0.37.0
```

Comment:
Question: is there a reason to upgrade ytt to 0.37.0? TKG BOM uses 0.38.0 and also we are using 0.38.0 in here https://github.com/vmware-tanzu/tanzu-framework/blob/main/hack/tools/Makefile#L29, should we upgrade it to 0.38.0 to keep it consistent across the project?

Reply:
Seems reasonable, any concerns @adobley ?

Reply:
I originally based it on the version of ytt that the kapp-controller we depend on in providers uses, but I'm happy to bump to 0.38.0.
I'll make those changes.